web3/_utils/module_testing/personal_module.py

```diff
@@ -343,3 +346,61 @@ def test_personal_sign_typed_data_deprecated(
             )
             assert signature == expected_signature
             assert len(signature) == 32 + 32 + 1
+
+
+class GoEthereumAsyncPersonalModuleTest:
+
+    @pytest.mark.asyncio
+    async def test_async_sign_and_ec_recover(self,
+                                             async_w3: "Web3",
+                                             unlockable_account_dual_type: ChecksumAddress,
+                                             unlockable_account_pw: str) -> None:
+        message = "This is a test"
+        signiture = await async_w3.geth.personal.sign(message,  # type: ignore
+                                                      unlockable_account_dual_type,
+                                                      unlockable_account_pw)
+        address = await async_w3.geth.personal.ec_recover(message, signiture)  # type: ignore
+        assert is_same_address(unlockable_account_dual_type, address)
+
+    @pytest.mark.asyncio
+    async def test_async_list_accounts(self, async_w3: "Web3") -> None:
+        accounts = await async_w3.geth.personal.list_accounts()  # type: ignore
+        assert len(accounts) > 0
+
+    @pytest.mark.asyncio
+    async def test_async_list_wallets(self, async_w3: "Web3") -> None:
+        wallets = await async_w3.geth.personal.list_wallets()  # type: ignore
+        assert isinstance(wallets[0], AttributeDict)
+
+    @pytest.mark.asyncio
+    async def test_async_new_account(self, async_w3: "Web3") -> None:
+        paraphrase = "Create New Account"
+        account = await async_w3.geth.personal.new_account(paraphrase)  # type: ignore
```

Comment:
passphrase

Reply:
wow, how many times can I spell this wrong?  haha.
fixed